tensorflow_lite_support/cc/task/processor/image_preprocessor.cc

```diff
@@ -68,6 +68,22 @@ absl::Status ImagePreprocessor::Init(
         "ImagePreprocessor only supports RGB color space for now.");
   }
 
+  // Determine if the input shape is resizable.
+  const TfLiteIntArray* dims_signature = Tensor()->dims_signature;
+
+  if (dims_signature != nullptr)
```

Comment:
Can you please format your code into the google style? See the [answer](https://stackoverflow.com/a/23408687) here about how to use clang-format.